tools/local-env/scripts/docker.js

```diff
@@ -7,5 +7,10 @@ dotenvExpand.expand( dotenv.config() );
 
 const composeFiles = local_env_utils.get_compose_files();
 
+if (process.argv.includes('--coverage-html')) {
+	process.env.LOCAL_PHP_XDEBUG = 'true';
+	process.env.LOCAL_PHP_XDEBUG_MODE = process.env.LOCAL_PHP_XDEBUG_MODE + ',coverage';
```

Comment:
```suggestion
	process.env.LOCAL_PHP_XDEBUG_MODE = 'coverage';
```

No need to preserve any other values for LOCAL_PHP_XDEBUG_MODE`.